dotnet/src/Connectors/Connectors.HuggingFace/TextGeneration/TextGenerationRequest.cs

```diff
@@ -1,23 +1,152 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+﻿
+// Copyright (c) Microsoft. All rights reserved.
 
 using System.Text.Json.Serialization;
 
-namespace Microsoft.SemanticKernel.Connectors.HuggingFace;
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Core;
 
 /// <summary>
-/// HTTP schema to perform completion request.
+/// HuggingFace text generation request object.
 /// </summary>
-public sealed class TextGenerationRequest
+internal sealed class TextGenerationRequest
 {
     /// <summary>
-    /// Prompt to complete.
+    /// A string to be generated from.
```

Comment:
`The input string to generate embeddings for.`

Reply:
This is not an embedding request, the description I took from their documentation:

![image](https://github.com/microsoft/semantic-kernel/assets/19890735/9f9bbab1-0dff-407a-a24e-9e031ad2e898)

Will update to your suggestion.